newm-server-common/src/main/kotlin/io/projectnewm/server/user/Password.kt

```diff
@@ -0,0 +1,15 @@
+package io.projectnewm.server.user
+
+import io.projectnewm.server.ext.toHash
+import io.projectnewm.server.ext.verify
+import io.projectnewm.server.serialization.PasswordSerializer
+import kotlinx.serialization.Serializable
+
+@Serializable(with = PasswordSerializer::class)
+data class Password(val value: String) {
+    override fun toString(): String = if (value.isBlank()) value else "***"
```

Comment:
Should the number of `*` be equivalent to the  length to the value?

Reply:
From a security perspective, it's better to not give any clue